### ISSUE TITLE
Determine GPS coordinates does not work

### ISSUE THREAD

**Issue by tessicz** [lmsgit/lms#1423]
Hello, in the current master branch does not work "Determine GPS coordinates automatically". In PHPUI I have set "googlemaps_api_key" and "gps_coordinate_providers" set to google. I see requests in https://console.cloud.google.com/apis/dashboard.
However, each request generates a bug and GPS coordinates are not returned to the LMS.

Can you advise me where the error is?

Thank you

**Comment by chilek** [comment]
1. For what kind of resource do you try to determine GPS coordinates automatically?
1. What kind of bugs request generates?
1. What can you see in Google Chrome developer tools js console?

**Comment by chilek** [comment]
Tried before moment to determine node/computer gps coordinates from my production installation using current master HEAD and it works!

**Comment by tessicz** [comment]
Generate testing for NODE and NETWORK DEVICE. There is no error in Google Chrome developer tools js console. As long as I did not fill out googlemaps_api_key and gps_coordinate_providers in PHPUI, it worked. Only the map wrote about the developer mode. So I've entered googlemaps API key. I created myself
  in a console.cloud.google.com account and created a new LMS project. in console.cloud.google.com we write the received request. What one request to one error. Unfortunately, console.cloud.google.com does not write a specific bug anywhere.

Are you trying to use googlemap_api_key?
